client/state/initial-state.js

```diff
@@ -48,7 +48,24 @@ function deserialize( state ) {
 	return reducer( state, { type: DESERIALIZE } );
 }
 
-function loadInitialState( initialState ) {
+function addSympathy( initialStateLoader ) {
+	if ( 'development' !== process.env.NODE_ENV || ( Math.random() > 0.5 ) ) {
```

Comment:
how should one go about doing this? is it easy to get access or should we use a command line flag?